src/backend/distributed/commands/vacuum.c

```diff
@@ -285,6 +291,9 @@ DeparseVacuumStmtPrefix(CitusVacuumParams vacuumParams)
 		&& vacuumParams.truncate == VACOPT_TERNARY_DEFAULT &&
 		vacuumParams.index_cleanup == VACOPT_TERNARY_DEFAULT
```

Comment:
Not really related to this PR, but since we are making this if statement even less readable than it already was lets improve the comping site
```suggestion
		&& vacuumParams.truncate == VACOPT_TERNARY_DEFAULT
		&& vacuumParams.index_cleanup == VACOPT_TERNARY_DEFAULT
```

Reply:
b0baa27